torchdynamo/debug_utils.py

```diff
@@ -101,12 +101,16 @@ def _cuda_system_info_comment():
         return "# torch.cuda.is_available()==False, no GPU info collected\n"
 
     model_str = "# CUDA Info: \n"
-    cuda_version_out = subprocess.run(["nvcc", "--version"], stdout=subprocess.PIPE)
-    cuda_version_lines = cuda_version_out.stdout.decode().split("\n")
-    cuda_version_out = "".join(
-        [f"# {s} \n" for s in cuda_version_lines if s not in [""]]
-    )
-    model_str += f"{cuda_version_out}\n"
+    try:
+        cuda_version_out = subprocess.run(["nvcc", "--version"], stdout=subprocess.PIPE)
+        cuda_version_lines = cuda_version_out.stdout.decode().split("\n")
+        cuda_version_out = "".join(
+            [f"# {s} \n" for s in cuda_version_lines if s not in [""]]
+        )
+        model_str += f"{cuda_version_out}\n"
+    except FileNotFoundError:
+        print("nvcc does not exist")
```

Comment:
Instead of printing here, can you write out "nvcc not found in path" to the model_str?